common/src/main/java/com/viaversion/viaversion/rewriter/ComponentRewriter.java

```diff
@@ -139,6 +140,42 @@ public void registerTabList(final C packetType) {
         });
     }
 
+    public void registerPlayerInfoUpdate1_20_3(final C packetType) {
+        protocol.registerClientbound(packetType, wrapper -> {
+            final BitSet actions = wrapper.passthrough(Types.PROFILE_ACTIONS_ENUM);
```

Comment:
Maybe add a `if (!actions.get(5)) return` here since you're not doing anything else with the packet? (Then you can also remove the check below)

Reply:
There can be multiple actions at once and we need to passthrough all bytes